### ISSUE TITLE
Let it run indefinitely

### ISSUE THREAD

**Issue by strites** [warner/wireguard-vanity-address#20]
Is there a way to let it run indefinitely? For the moment, it stops running after an arbitrary time - even without finding any key

**Comment by andrewdunndev** [comment]
I'll glom on here, I was going to use a good deal of electricity for vanity on a longer address (eight characters) and it appears to stop after a handful of seconds.

**Comment by warner** [comment]
yeah, that's definitely a good idea, I think I was being overly cautious when I first wrote that loop, but turning it into a simple infinite loop is a fine idea

**Comment by jellium** [comment]
How would one tweak the loop at the end of the main() function in order to have an infinite loop?

**Comment by martinvonwittich** [comment]
> How would one tweak the loop at the end of the main() function in order to have an infinite loop?

Replacing the `(0..100_000_000)` with `rayon::iter::repeat(0)` has worked for me.

**Comment by Sveske-Juice** [comment]
A simple workaround in bash is to just make a oneline infinite while loop:
```bash
while :; do <command>; done
```
